aspnetcore/release-notes/aspnetcore-5.0.md

```diff
@@ -0,0 +1,476 @@
+
+Blazor WebAssembly apps target the full .NET API surface area, but not all .NET APIs are supported on WebAssembly due to browser sandbox constraints. Unsupported APIs throw <xref:System.PlatformNotSupportedException> when running on WebAssembly. A platform compatibility analyzer warns the developer when the app uses APIs that aren't supported by the app's target platforms. For more information, see <xref:blazor/components/class-libraries#browser-compatibility-analyzer-for-blazor-webassembly>.
+
+### Lazy load assemblies
+
+Blazor WebAssembly app startup performance can be improved by deferring the loading of some application assemblies until they are required. For more information, see <xref:blazor/webassembly-lazy-load-assemblies>.
+
+### Updated globalization support
+
+Globalization support is available for Blazor WebAssembly based on International Components for Unicode (ICU). For more information, see <xref:blazor/globalization-localization>.
+
+## gRPC
+
+Many preformance improvements have been made in [gRPC](https://grpc.io/). <!-- For more information, see [this ]
+Waiting for https://github.com/aspnet/specs/pull/523
+--> 
+For more information, see <xref:grpc/index>.
+
+## SignalR
+
+SignalR Hub filters, called Hub pipelines in ASP.NET SignalR, is a feature that allows code to run before and after Hub methods are called. Running code before and after Hub methods are called is similar to how middleware has the ability to run code before and after an HTTP request. Common uses include logging, error handling, and argument validation.
+
+For more information, see [Use hub filters in ASP.NET Core SignalR](xref:signalr/hub-filters).
+
+### SignalR parallel hub invocations
+
+ASP.NET Core SignalR is now capable of handling parallel hub invocations. The default behavior can be changed to allow clients to invoke more than one hub method at a time:
+
+[!code-csharp[](~/release-notes/sample/StartupSignalRhubs.cs?name=snippet)]
+
+### Added Messagepack support in SignalR Java client
+
+A new package, [com.microsoft.signalr.messagepack](https://search.maven.org/search?q=g:com.microsoft.signalr.messagepack), adds MessagePack support to the SignalR Java client. To use the MessagePack hub protocol, add `.withHubProtocol(new MessagePackHubProtocol())` to the connection builder:
+
+```java
+HubConnection hubConnection = HubConnectionBuilder.create(
+                           "http://localhost:53353/MyHub")
+               .withHubProtocol(new MessagePackHubProtocol())
+               .build();
+```
+
+<!--
+See [Update SignalR code](xref:migration/31-to-50#signalr) for migration instructions.
+-->
+
+## Kestrel
+
+* Reloadable endpoints via configuration: Kestrel can detect changes to configuration passed to [KestrelServerOptions.Configure](xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.Configure%2A) and unbind from existing endpoints and bind to new endpoints without requiring an application restart.
```

Comment:
I think we should point out that the "Kestrel" configuration subsection is bound by default when you call `.ConfigureWebHostDefaults(...` like our default templates do. Maybe something like the following.

```suggestion
* Reloadable endpoints via configuration: Kestrel can detect changes to configuration passed to [KestrelServerOptions.Configure](xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.Configure%2A) and unbind from existing endpoints and bind to new endpoints without requiring an application restart. By default when using <xref:Microsoft.Extensions.Hosting.GenericHostBuilderExtensions.ConfigureWebHostDefaults%2A> or <xref:Microsoft.AspNetCore.WebHost.CreateDefaultBuilder%2A>, Kestrel binds to the "Kestrel" configuration subsection.
```